### ISSUE TITLE
webgl_threejs can't display model correctly.

### ISSUE THREAD

**Issue by liuxin2322** [tpaviot/pythonocc-core#645]
in web brower,model can't be displayed correctly. some surface are missing.
![360截图20190316194212365](https://user-images.githubusercontent.com/42459125/54474808-93f76f80-4824-11e9-9548-7dc7c5ec85d0.jpg)
i confirm the model is right because it was tested and displayed entirely in Qt.

![jjj](https://user-images.githubusercontent.com/42459125/54475254-37974e80-482a-11e9-8db0-c4cde95a6aad.jpg)

source code :
```python
from OCC.Extend.DataExchange import read_iges_file
from OCC.Display.WebGl import threejs_renderer

big_shp = read_iges_file("j3.igs")

my_renderer = threejs_renderer.ThreejsRenderer()
my_renderer.DisplayShape(big_shp,color=(1,1,0))
my_renderer.render()

```

**Comment by tpaviot** [comment]
I need the igs file to test

**Comment by liuxin2322** [comment]
> I need the igs file to test

ok,here. thank you in advance

[j3.zip](https://github.com/tpaviot/pythonocc-core/files/2974850/j3.zip)

**Comment by tpaviot** [comment]
Issue confirmed.

**Comment by tpaviot** [comment]
which os do you run ?

**Comment by liuxin2322** [comment]
> which os do you run ?

win10，pythonocc0.18.2

**Comment by tpaviot** [comment]
The issue seems to be related to the threejs renderer only. On my machine (Ubuntu 18.04), the x3dom renderer works as expected. Can you please try

**Comment by liuxin2322** [comment]
> The issue seems to be related to the threejs renderer only. On my machine (Ubuntu 18.04), the x3dom renderer works as expected. Can you please try

you're right,x3dom works well.  is there any solutions about threejs?

**Comment by tpaviot** [comment]
I got the threejs renderer working by forcing double sided rendering. You can force double siding by modifying the index.html file, and adding the property ```side: THREE.DoubleSide``` to the MeshPhongMaterial.

**Comment by tpaviot** [comment]
Fixed by commit d6a4623532182bd9cdfc84455b690e3f458e91b4
